var/spack/repos/builtin/packages/icebin/package.py

```diff
@@ -0,0 +1,96 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Icebin(CMakePackage):
+    """Regridding/Coupling library for GCM + Ice Sheet Model"""
+
+    homepage = "https://github.com/citibeth/icebin"
+    url = "https://codeland.github.com/citibeth/icebin/tar.gz/v0.1.0.tar.gz"
+
+    maintainers = ['citibeth']
+
+    version('0.1.5', 'a209c4cd4502b9ded7525ebaa7c6107f')
+    version('0.1.4', '5c8ecd778255f2972c0caa3a9f5d82ea')
+    version('0.1.3', '98d9b28ef8f8a145a7eed2cb1e92e7e4')
+    version('0.1.2', '68673158b46b6e88aea6bc4595444adb')
+    version('0.1.1', '986b8b51a2564f9c52156a11642e596c')
+    version('0.1.0', '1c2769a0cb3531e4086b885dc7a6fd27')
+
+    version('develop',
+        git='https://github.com/citibeth/icebin.git',
+        branch='develop')
+
+    variant('python', default=True, description='Build Python extension (requires Python, Numpy)')
+    variant('gridgen', default=True, description='Build grid generators (requires CGAL, GMP, MPFR)')
+    variant('coupler', default=False, description='Build the GCM couplers (requires MPI)')
```

Comment:
We usually default to `+mpi` (mostly for broken concretizer reasons). Do you have any strong feelings about this?

Reply:
No strong feelings.  But since there is no `mpi` variant, I'm not sure what I would do?

Reply:
I'm just wondering if we should default to `+coupler` since it implies MPI support. But I don't have any strong opinions either. I don't think it will cause any concretization problems.